website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,188 @@
+
+#### Footer
+
+By default, the footer displays below the options list and contains content and/or buttons which provide additional context or functions related to the options.
+
+##### Single
+
+The footer for the Single component includes a total count, which is customizable to fit the context.
+
+![Custom content enabled before the list in both Single and Multiple components](/assets/components/form/super-select/super-select-single-variant-footer-with-counter.png)
+
+##### Multiple
+
+The footer for the Multiple component includes buttons to modify the options and a count of the number of options selected out of the total.
+
+![Custom content enabled before the list in both Single and Multiple components](/assets/components/form/super-select/super-select-multiple-variant-footer.png)
+
+The “Show selected” button displays only the selected options in the list. When toggled, the text changes to “Show all.”
+
+![Flow showing the behavior of the show selected button and how it toggles the list to only display selected list items](/assets/components/form/super-select/super-select-flow-select-all.png)
+
+If “Show selected” is toggled when no options have been selected, a message will appear notifying the user.
+
+![Flow showing the behavior of the show selected button when no selections have been made yet.](/assets/components/form/super-select/super-select-flow-no-selections-select-all.png)
+
+The “Clear selected” button clears all selections and reverts the Super Select to its default/deselected state. This button is hidden if no options are selected.
+
+![Flow showing the behavior of the clear selected button and how it reverts the Super Select's state to the default one](/assets/components/form/super-select/super-select-flow-clear-selected.png)
+
+##### Footer with custom content
+
+The footer can contain custom content instead of the default content. This is helpful in maintaining the standard appearance of the footer container while allowing for custom usage.
+
+![Showing the footer with custom content inside.](/assets/components/form/super-select/super-select-single-multiple-footer-custom-content.png =836x*) 
+
+#### CustomContent
+
+If enabled, custom content will occupy this space.
+
+![Showing the afterOptions slot with custom content](/assets/components/form/super-select/super-select-after-options-custom-content.png =836x*)
+
+## List
+
+### Positioning
+
+Lists can be positioned to the left or right, as well as above or below the trigger as necessary to fit within the UI. These options are only available when `matchTriggerWidth` is set to `false`. This effects the ember component only.
+
+![Showing the different list positions (left, right + top and bottom)](/assets/components/form/super-select/super-select-list-positioning.png =836x*)
+
+### Width
```

Comment:
Should we provide examples of these use cases (width/auto-width/max-width) in the "how to use" section?

Reply:
I believe this would be a question for @KristinLBradley?

Reply:
I can. I had considered these to be less commonly needed options so hadn't included examples originally.

I'll see if I can come with some useful examples though.

Reply:
I was seeing some issues as I was adding examples that I'll investigate...

Reply:
Fixed issues I was seeing in component branch and added an example on "Limiting width" to "How to use" section.